libs/client/LightEpoch.cs

```diff
@@ -344,14 +344,14 @@ int ComputeNewSafeToReclaimEpoch(int currentEpoch)
         {
             int oldestOngoingCall = currentEpoch;
 
-            for (int index = 1; index <= kTableSize; ++index)
+            for (int index = 1; index <= TableSize; ++index)
             {
-                int entry_epoch = (*(tableAligned + index)).localCurrentEpoch;
-                if (0 != entry_epoch)
+                int entryEpoch = (*(tableAligned + index)).localCurrentEpoch;
+                if (0 != entryEpoch)
```

Comment:
```suggestion
                if (entryEpoch != 0)
```

Reply:
Thank you for the good tips,
I solved it